keyboards/atlantis/ak7/keymaps/default/keymap.c

```diff
@@ -0,0 +1,108 @@
+// Copyright 2021 Atlantis (78452063@qq.com)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "ak7.h"
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	[0] = LAYOUT(
+		KC_VOLD, MO(1), KC_VOLU,
+		LCTL(KC_C), LCTL(KC_X), LCTL(KC_V),
+		KC_F1, KC_F2, KC_RSFT),
+
+	[1] = LAYOUT(
+		KC_A, KC_TRNS, KC_B,
+		RGB_TOG, RGB_MOD, KC_TRNS,
+		KC_TRNS,KC_TRNS,KC_TRNS)
+};
+
+
+bool is_alt_tab_active = false; // ALT_TAB激活状态。
```

Comment:
Could these comments be translated to English?

Reply:
From what I can see the variable names are pretty self evident, and the comments are just for what the variables do

Reply:
Then perhaps the comments aren't needed at all.